### PR TITLE
28: Convert Sass files to partials, conform to USDS, use local images.

### PR DESCRIPTION
**Pull request by bixgomez**
This PR represents the work done for https://github.com/nerdstein/nerdstein-design-library/issues/28

### CODE REVIEW

File: html/css/pattern-scaffolding.css [hunk 1]
```diff
@@ -1,54 +0,0 @@
-/**
```
Comment (nerdstein) [inline html/css/pattern-scaffolding.css:1]: Is there a reason why this file has been removed? I thought Pattern Lab needed it

File: html/patterns/01-atoms-02-list-body-list-body/01-atoms-02-list-body-list-body.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
     
-    <link rel="stylesheet" href="../../css/style.css?1497365795" media="all" />
-    <link rel="stylesheet" href="../../css/pattern-scaffolding.css?1497365795" media="all" />
+    <link rel="stylesheet" href="../../css/style.css?1508604484" media="all" />
+    <link rel="stylesheet" href="../../css/pattern-scaffolding.css?1508604484" media="all" />
```
Comment (nerdstein) [inline html/patterns/01-atoms-02-list-body-list-body/01-atoms-02-list-body-list-body.html:9]: So, the HTML files seem to still be expecting this CSS file but it was removed above. There also seems to be a bunch of other files removed. I'm curious if this was intentional

File: html/patterns/01-atoms-08-something-else-something-else/01-atoms-08-something-else-something-else.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (nerdstein) [inline html/patterns/01-atoms-08-something-else-something-else/01-atoms-08-something-else-something-else.html:1]: Is "something else" a pattern you added? Was it a test? If so, please remove it and work on the patterns in the issue queue.

**Review by nerdstein — CHANGES_REQUESTED** [review]
@bixgomez please take a look at my feedback, and thanks for taking a look

**Comment by nerdstein** [comment]
@bixgomez also note that there is a pull request template. were you asked to fill it out or is it broken?

**Comment by bixgomez** [comment]
Closed because I opened a new one on a unique branch.

**Pull request closed without merging**